dockstore-webservice/src/main/java/io/dockstore/webservice/core/Service.java

```diff
@@ -25,7 +26,13 @@
 @ApiModel(value = "Service", description = "This describes one service in the dockstore as a special degenerate case of a workflow", parent = Workflow.class)
 @Entity
 @Table(name = "service")
-@NamedQuery(name = "io.dockstore.webservice.core.Service.findAllPublishedPaths", query = "SELECT new io.dockstore.webservice.core.database.WorkflowPath(c.sourceControl, c.organization, c.repository, c.workflowName) from Service c where c.isPublished = true")
+@NamedQueries({
+        @NamedQuery(name = "io.dockstore.webservice.core.Service.findAllPublishedPaths", query = "SELECT new io.dockstore.webservice.core.database.WorkflowPath(c.sourceControl, c.organization, c.repository, c.workflowName) from Service c where c.isPublished = true"),
+        @NamedQuery(name = "io.dockstore.webservice.core.Service.getEntryLiteByUserId", query = "SELECT new io.dockstore.webservice.core.database.EntryLite$EntryLiteService(c.sourceControl, c.organization, c.repository, c.workflowName, c.dbUpdateDate as entryUpdated, MAX(v.dbUpdateDate) as versionUpdated) "
+                + "FROM Service c LEFT JOIN c.workflowVersions v "
```

Comment:
Optional nitpick -- would call `s` instead of `c`.

Reply:
the c pattern is in all named queries for tools and workflows too so I just carried it over, not sure why its c though? Container?

Seems like we don't have things set up test named queries directly, the tests for postgres can only query in SQL. Denis suggested a an integration test and the existing test already calls the endpoint and checks the path generated. I could make a test using the generated sql, but that seemed a bit indirect. Open to other ideas.

Reply:
The short name is a stand-in for the longer name, so I like it to reflect the table (or I guess in this case entity) name. I did say optional, and I don't feel too strongly about it.

If the existing integration tests validate the endpoint, I think that's fine; just wanted to make sure.

Reply:
The c is back from when we called tools containers, we should try to replace it when we can.

Reply:
Yup